rel/i18n/emqx_ds_schema.hocon

```diff
@@ -0,0 +1,35 @@
+emqx_ds_schema {
+
+messages.desc:
+"""Configuration related to the durable storage of MQTT messages."""
+
+builtin.desc:
+"""Builtin session storage backend utilizing embedded RocksDB key-value store."""
+
+builtin_data_dir.desc:
+"""File system directory where the database is located."""
```

Comment:
it has no default value in the schema. 
so it's necessary to describe from where the default value would be inherited if missing from the config.